protocol-test-util/src/main/java/io/camunda/zeebe/test/broker/protocol/commandapi/PartitionTestClient.java

```diff
@@ -230,7 +230,7 @@ public void updateVariables(
             .command()
             .put("scopeKey", scopeKey)
             .put("updateSemantics", updateSemantics)
-            .put("document", MsgPackUtil.asMsgPack(document).byteArray())
+            .put("variables", MsgPackUtil.asMsgPack(document).byteArray())
```

Comment:
Just a little mistake I noticed - we renamed the property at some point but never updated it here.